### DISCUSSION TITLE
Share database with overpass turbo

### DISCUSSION THREAD

**Discussion by alvasilvao** [mediagis/nominatim-docker#465] [Q&A]
Hi,
Is it possible to download the planet file (independently) and then have nominatim docker use it?
I am asking because I my use case requires the use of both nominatim and overpass turbo and I would onviously like to download the planet file only one.
Is there any other approach I should consider?
Thanks!

**Comment by mtmail** [comment]
Yes, a planet file is just like an extract (that contains 100% of OSM data). From the documentation 

```
PBF_URL: Which [OSM extract](https://github.com/mediagis/nominatim-docker/tree/master/4.2#openstreetmap-data-extracts) to download and import. It cannot be used together with PBF_PATH. Check https://download.geofabrik.de/
PBF_PATH: Which [OSM extract](https://github.com/mediagis/nominatim-docker/tree/master/4.2#openstreetmap-data-extracts) to import from the .pbf file inside the container. It cannot be used together with PBF_URL.
```

**Reply by alvasilvao** [reply]
Thank you for your answer! I had seen that in the documentation, but I wasn't sure what it meant.
Thanks again

**Reply by alvasilvao** [reply]
Hi again,
I read more about the formats and the question I have is actually whether you can point nominatim to an existing database instead of a pbf file. What I am trying to avoid is that each docker creates its own database and therefore ending up with several full planet OSM databases.
Thanks!

**Reply by mtmail** [reply]
That’s now possible. Overpass doesn’t use a (postgresql) database https://wiki.openstreetmap.org/wiki/Overpass_API/Installation
